src/test/scala/au/com/agiledigital/toolform/command/generate/kubernetes/minikube/GenerateMinikubeTest.scala

```diff
@@ -57,4 +59,24 @@ class GenerateMinikubeTest extends FlatSpec with Matchers with PrivateMethodTest
     val result          = ToolFormAppSimulator.simulateAppForTest(List("generate", "minikube", "-i", inputFile.getAbsolutePath, "-o", "/tmp/foo/bar/baz/generate-docker-test.out").toArray)
     result should startWith("Output directory [/tmp/foo/bar/baz] does not exist")
   }
+
+  "isDiskResourceType" should "should return true if resource is disk type" in {
+    val testResource: Resource = {
+      val testResource = new Resource("ID", "disk", None, storage = None)
+      testResource
```

Comment:
Not really too important because it's just a test but you don't need `new` to construct the object in Scala. Also I think you could even lose the braces here.
```
val testResource = Resource("ID", "disk", None, storage = None)
```

Reply:
Excellent, good to know. I've cleaned instances of those up in this file and in `VolumeClaimWriterTest.scala`